addon/mongocrypt.h

```diff
@@ -56,6 +56,8 @@ namespace opensslcrypto {
 std::unique_ptr<CryptoHooks> createOpenSSLCryptoHooks();
 }
 
+typedef bool (*ExplicitEncryptionContextInitFunction)(mongocrypt_ctx_t*, mongocrypt_binary_t*);
```

Comment:
For my own understanding, this is declaring a type, `ExplicitEncryptionContextInitFunction` that takes in a `mongocrypt_ctx_t*` and `mongocrypt_binary_t*` and returns a `bool`, correct?

Reply:
It declares a type alias, not a new type, but otherwise yup 👍